encode/av1encode.c

```diff
@@ -875,7 +887,43 @@ static int init_va(void)
     va_status = vaInitialize(va_dpy, &major_ver, &minor_ver);
     CHECK_VASTATUS(va_status, "vaInitialize");
     av1_profile = VAProfileAV1Profile0;
-    entryPoint = VAEntrypointEncSliceLP;
+
+    // select entrypoint
+
+    int num_entrypoints = vaMaxNumEntrypoints(va_dpy);
+    VAEntrypoint* entrypoints = malloc(num_entrypoints * sizeof(*entrypoints));
+    if (!entrypoints) {
+        fprintf(stderr, "error: failed to initialize VA entrypoints array\n");
+        exit(1);
+    }
+
+    vaQueryConfigEntrypoints(va_dpy, av1_profile, entrypoints, &num_entrypoints);
+    int support_encode = 0;
```

Comment:
Can be bool?

Reply:
It can be bool. But I will just follow the coding style with other codecs.